build.gradle.kts

```diff
@@ -1,41 +1,62 @@
 plugins {
     `kotlin-dsl`
     kotlin("jvm") version embeddedKotlinVersion
-    id("com.jfrog.bintray") version "1.8.5"
     `maven-publish`
+    signing
+    id("org.jetbrains.dokka") version "1.4.20"
 }
 
 group = "com.wantedly"
 version = "1.0.0-SNAPSHOT"
 
 repositories {
+    mavenCentral()
     jcenter()
```

Comment:
❓ Can this be removed?

Reply:
We can't remove this because Dokka depends on `jcenter()`.
https://github.com/Kotlin/dokka/issues/41